src/main/java/org/fundacionjala/coding/richardseverich/Movies/Customer.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.coding.richardseverich.Movies;
+
+import java.util.ArrayList;
+
+/**
+ * Created by RichardSeverich on 5/31/2017.
+ */
+public class Customer {
+    private String nameCustomer;
+    private ArrayList<Rental> rentalList;
+    private int totalAmount;
+    private int frequentRenterPoints;
+
+    /**
+     * @param name is name of costumer.
+     */
+    public Customer(String name) {
+        nameCustomer = name;
+        rentalList = new ArrayList<>();
+        totalAmount = 0;
+        frequentRenterPoints = 0;
```

Comment:
you don't initialize these integer values
by default Integer attributes of class are initialized in 0